railties/lib/rails/generators/rails/app/templates/config/initializers/hsts_subdomains.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.action_dispatch.hsts_subdomains = true
```

Comment:
@rafaelfranca That will mess up with existing ssl_options present in `config/environments/production.rb` right?